java/arcs/core/crdt/CrdtSet.kt

```diff
@@ -21,17 +21,27 @@ import arcs.core.crdt.internal.Actor
 import arcs.core.crdt.internal.VersionMap
 
 /** A [CrdtModel] capable of managing a set of items [T]. */
-class CrdtSet<T : Referencable> @JvmOverloads constructor(
+class CrdtSet<T : Referencable> (
     /** Initial data. */
     /* internal */ var _data: Data<T> = DataImpl(),
     /** Function to construct a new, empty [Data] object with a given [VersionMap]. */
-    private val dataBuilder: (VersionMap) -> Data<T> = { DataImpl(it) }
-) : CrdtModel<Data<T>, CrdtSet.IOperation<T>, Set<T>> {
+    private val dataBuilder: (VersionMap) -> Data<T>// = { DataImpl(it) }
+    ) : CrdtModel<Data<T>, CrdtSet.IOperation<T>, Set<T>> {
```

Comment:
Indentation on this line is 4-spaces too much.

Reply:
done